backend/src/database/entities/bcws/bcws-personnel.entity.ts

```diff
@@ -0,0 +1,219 @@
+
+  @Column({
+    name: 'liason_phone_number',
+    type: 'varchar',
+    length: 10,
+    nullable: true,
+  })
+  liaisonPhoneNumber?: string;
+
+  @Column({ name: 'liason_email', type: 'varchar', length: 50, nullable: true })
+  liaisonEmail?: string;
+
+  @Column({ name: 'coordinator_notes', type: 'text', nullable: true })
+  coordinatorNotes?: string;
+
+  @Column({ name: 'logistics_notes', type: 'text', nullable: true })
+  logisticsNotes?: string;
+
+  @Column({ name: 'willingess_statement', type: 'boolean', default: false })
+  willingnessStatement: boolean;
+
+  @Column({ name: 'par_q', type: 'boolean', default: false })
+  parQ: boolean;
+
+  @Column({ name: 'workplace_policy', type: 'boolean', default: false })
+  respectfulWorkplacePolicy: boolean;
+
+  @Column({ name: 'orientation', type: 'boolean', default: false })
+  orientation: boolean;
+
+  @Column({
+    name: 'highest_ofa',
+    type: 'enum',
+    enum: OFA,
+    enumName: 'ofa',
+    nullable: true,
+  })
+  highestOFA?: OFA;
+
+  @Column({ name: 'ofa_expiry', type: 'timestamp', nullable: true })
+  ofaExpiryDate?: Date;
+
+  @Column({ name: 'food_safety_I', type: 'boolean', default: false })
+  foodSafetyLevelI?: boolean;
+
+  @Column({ name: 'food_safety_I_expiry', type: 'timestamp', nullable: true })
+  foodSafetyCertificationLevelIExpiry?: Date;
+
+  @Column({ name: 'food_safety_II', type: 'boolean', default: false })
+  foodSafetyLevelII?: boolean;
+
+  @Column({ name: 'food_safety_II_expiry', type: 'timestamp', nullable: true })
+  foodSafetyCertificationLevelIIExpiry?: Date;
+
+  @Column({
+    name: 'second_choice_role',
+    type: 'enum',
+    enumName: 'bcws-role',
+    enum: BcwsRole,
+  })
+  secondChoiceRole: BcwsRole;
+
+  @Column({
+    name: 'first_choice_role',
+    type: 'enum',
+    enumName: 'bcws-role',
+    enum: BcwsRole,
+  })
+  firstChoiceRole: BcwsRole;
+
+  @OneToMany(() => BcwsPersonnelTools, (b) => b.personnel, { cascade: true })
+  tools?: BcwsPersonnelTools[];
+
+  @OneToMany(() => BcwsPersonnelCertificationEntity, (c) => c.personnel, {
+    cascade: true,
+  })
+  certifications: BcwsPersonnelCertificationEntity[];
+
+  @OneToMany(() => BcwsSectionsAndRolesEntity, (s) => s.personnel, {
+    cascade: true,
+  })
+  roles?: BcwsSectionsAndRolesEntity[];
+
+  @OneToMany(() => LanguageEntity, (l) => l.personnel, { cascade: true })
+  languages?: LanguageEntity[];
+
+  toResponseObject(role: Role, lastDeployed?: string): Record<string, BcwsRO> {
+    const response = new BcwsRO();
+
+    const personnelData = this.personnel.toResponseObject(role, lastDeployed);
+    const data = {
+      ...personnelData,
+      homeFireCentre: this?.homeFireCentre?.toResponseObject() ?? {},
+      workFireCentre: this?.workFireCentre?.toResponseObject() ?? {},
+      employeeId: this.employeeId,
+      paylistId: this.paylistId,
+      dateApplied: this.dateApplied,
+      dateApproved: this.dateApproved,
+      coordinatorNotes: this.coordinatorNotes,
```

Comment:
Wondering if there's a scenario that will warrant renaming this to `bcwsCoordinatorNotes` and `bcwsLogisticsNotes`. Whether just in the RO or the entity itself. Not seeing it immediately, so if you don't see it, feel free to disregard this comment.

Reply:
I was torn on this one. If we use the same key name for the values which have the same key for both tables, but different values we can just keep them in the personnelRO which each program extends, which I think will result in less changes for the current places these keys are referenced. However, I could also see a case for renaming the keys of these fields.

Reply:
We can keep for now, easy to change in the future